### ISSUE TITLE
Write a function that returns returns tape name and object store name

### ISSUE THREAD

**Issue by nmassey001** [cedadev/nlds#94]
Currently the code "munges" the name of the file to get the tape path and object store path in multiple places.
It would be much better if we could write a function that does this munging so there is no chance of making a mistake in the code.

e.g.:
```
          holding_tape_path = (f"root://{tape_server}/{tape_base_dir}/"
                                 f"{holding_slug}")
          full_tape_path = (f"{holding_tape_path}/{tar_filename}")
```

and:

```
            bucket_name, object_name = (
                   path_details.object_name.split(':')
            )
```

and even worse in the delete code router (my doing)!

**Comment by nmassey001** [comment]
- [ ] object name function
- [ ] tape name function
- [ ] use in put_transfer
- [ ] use in get_transfer
- [ ] use in del_transfer
- [ ] use in archive_put
- [ ] use in archive_get

**Comment by nmassey001** [comment]
Fixed and verified in v1.0.2
